Makefile

```diff
@@ -85,10 +85,8 @@ LDFLAGS = -X pkg/version/info.Version=$(VERSION) -X pkg/awsutils/awssession.vers
 ALLPKGS = $(shell go list $(VENDOR_OVERRIDE_FLAG) ./... | grep -v cmd/packet-verifier)
 # BINS is the set of built command executables.
 BINS = aws-k8s-agent aws-cni grpc-health-probe cni-metrics-helper aws-vpc-cni aws-vpc-cni-init egress-v4-cni
-# Plugin binaries
-# Not copied: bridge dhcp firewall flannel host-device host-local ipvlan macvlan ptp sbr static tuning vlan
-# For gnu tar, the full path in the tar file is required
-PLUGIN_BINS = ./loopback ./portmap ./bandwidth ./host-local
+# CORE_PLUGIN_DIR is the directory containing upstream containernetworking plugins
+CORE_PLUGIN_DIR = $(MAKEFILE_PATH)/core-plugins/
```

Comment:
Do we end up installing all these plugins - 

<img width="319" alt="Screenshot 2023-04-24 at 11 55 23 AM" src="https://user-images.githubusercontent.com/1111446/234090031-0a815fe0-8ab9-4aaa-aabc-6f503d219cc2.png">

Reply:
Yep, all of those are installed. I chose not to specify them specifically in an attempt to make this more future-proof